menoh/CMakeLists.txt

```diff
@@ -38,6 +37,12 @@ target_link_libraries(menoh_chainer_compiler
   absl::optional
   )
 
+if (!WIN32)
+  target_link_libraries(menoh_chainer_compiler
+  pthread
```

Comment:
I think we should define `${CHAINER_COMPILER_PTHREAD_LIBRARIES}` or something in the toplevel CMakeFiles.txt